compiler/src/iree/compiler/Codegen/Common/DecomposeBatchMmt4DOps.cpp

```diff
@@ -60,7 +61,23 @@ struct ConvertBatchMmt4DtoMmt4DPattern
     auto outType = out.getType().cast<RankedTensorType>();
     // Batch dim needs to be tiled to 1 first.
     if (outType.getShape()[0] != 1) {
-      return rewriter.notifyMatchFailure(op, "batch dim needs to be 1");
+      SmallVector<int64_t> tileSizes{1};
+      auto tilingInterfaceOp = cast<TilingInterface>(op.getOperation());
+      scf::SCFTileAndFuseOptions options{};
+      options.setTilingOptions(scf::SCFTilingOptions().setTileSizes(
+          getAsIndexOpFoldResult(op.getContext(), tileSizes)));
+      FailureOr<scf::SCFTileAndFuseResult> tiledResults =
+          scf::tileConsumerAndFuseProducerGreedilyUsingSCFForOp(
+              rewriter, tilingInterfaceOp, options);
+      if (failed(tiledResults))
```

Comment:
Nit: Add `{` `}` around even single line statements (IREE diverges from LLVM style in this one way).